index.js

```diff
@@ -58,7 +58,8 @@ def('Node',
     [$.Any, $Node],
     _node => ({
       '_@@type': 'sanctuary-html/Node',
-      equals: _other => $Node.test(_other) && String(_other) === String(_node),
+      equals: other => $Node.test(other) &&
+              _html(other.value) === _html(_node),
```

Comment:
I believe this fixes a bug in our original implementation. Two `Node`'s are equal iff they have the same HTML representation. Casting to `String` just gave us `[object Object]`.